gamemodes/terrortown/gamemode/shared/lang/german.lua

```diff
@@ -38,8 +38,8 @@ L.win_innocents = "Die Innos haben gewonnen!"
 L.win_bees = "Die Bienen haben gewonnen! (Es ist ein Unentschieden)"
 L.win_showreport = "Schauen wir uns den Rundenbericht die nächste(n) {num} Sekunde(n) an."
 
-L.limit_round = "Rundenlimit erreicht. {mapname} wird bald geladen."
-L.limit_time = "Zeitlimit erreicht. {mapname} wird bald geladen."
+L.limit_round = "Rundenlimit erreicht. Die nächste Map wird bald geladen."
```

Comment:
Why doing this? Fixing this here seems to be inconsistent. If mapname is displayed cryptically, this is a fault on the lua side.

Reply:
It is often impossible to tell the next mapname beforehand. It's more consistent, since as of now it will always be wrong if you are using a mapvote addon instead of the default map cycling, which is atleast in my perception the case on the vast majority of servers. See this [PR](https://github.com/TTT-2/TTT2/pull/400).